pkg/store/filenames/filenames.go

```diff
@@ -45,8 +45,11 @@ const (
 	VzIdentifier       = "vz-identifier"
 	VzEfi              = "vz-efi"
 
-	// SocketDir is the default location for forwarded sockets with a relative paths in HostSocket
-	SocketDir = "sock"
+	// InstanceSocketDir is the default location for forwarded sockets with a relative path in HostSocket
+	InstanceSocketDir = "sock"
+
+	// InstanceConfigDir is the default location for config files with a relative path in HostFile
+	InstanceConfigDir = "conf"
```

Comment:
I still think this name is confusing.

What about "copied-from-guest"

Reply:
Done